mlflow/pytorch/__init__.py

```diff
@@ -593,6 +634,31 @@ def _load_model(path, **kwargs):
             return torch.jit.load(model_path)
 
 
+@experimental
+def load_state_dict(model_uri):
+    """
+    Load a state dict from a local file or a run.
+
+    :param model_uri: The location, in URI format, for example:
+
+                    - ``/Users/me/path/to/local/model``
+                    - ``relative/path/to/local/model``
+                    - ``s3://my_bucket/path/to/model``
+                    - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
+
+                    For more information about supported URI schemes, see
+                    `Referencing Artifacts <https://www.mlflow.org/docs/latest/concepts.html#
+                    artifact-locations>`_.
+    :return: A pytorch state dict
+    """
+    import torch
+
+    local_model_path = _download_artifact_from_uri(artifact_uri=model_uri)
+    model_path = os.path.join(local_model_path, _TORCH_STATE_DICT_FILE_NAME)
+    state_dict = torch.load(model_path)
+    return state_dict
```

Comment:
Can we replace "model" with "state_dict" in this function to avoid confusion and clarify what this function expects is not the URI of an MLflow model?